api/handlers_test.go

```diff
@@ -1720,6 +1720,411 @@ func TestServer_JetDropsByJetID_NextPrevTests(t *testing.T) {
 	})
 }
 
+func TestServer_JetDropsByJetID_NextPrevTests_Siblings(t *testing.T) {
+	defer testutils.TruncateTables(t, testDB, []interface{}{models.Record{}, models.JetDrop{}, models.Pulse{}})
+	totalCount := 7
+	// making different siblings in same pulses
+	_, preparedJetDrops1, preparedPulses := testutils.GenerateJetDropsWithSomeJetID(t, totalCount)
+	_, preparedJetDrops2, _ := testutils.GenerateJetDropsWithSomeJetID(t, totalCount)
+	_, preparedJetDrops3, _ := testutils.GenerateJetDropsWithSomeJetID(t, totalCount)
+	_, preparedJetDrops4, _ := testutils.GenerateJetDropsWithSomeJetID(t, totalCount)
+	for i := 0; i <= len(preparedJetDrops1)-1; i++ {
+		preparedJetDrops1[i].PulseNumber = preparedPulses[i].PulseNumber
+		preparedJetDrops2[i].PulseNumber = preparedPulses[i].PulseNumber
+		preparedJetDrops3[i].PulseNumber = preparedPulses[i].PulseNumber
+		preparedJetDrops4[i].PulseNumber = preparedPulses[i].PulseNumber
+
+		if i-1 >= 0 {
```

Comment:
if i >= 1  )) it's ok. compiles will change it. you can leave it here

Reply:
it's more obvious